### DISCUSSION TITLE
如何获取点击的对象？

### DISCUSSION THREAD

**Discussion by huangzhtao** [antvis/G2#5569] [Q&A]
想实现一个功能，通过点击图形上的元素获取对象，然后可以调整对象的属性。
通过监听plot、element或者plot的点击事件可以获取事件相关属性，例如data、target等。
有几个问题：

1. 如何获取点击的对象，获取到chart、interval、axis等，以便获取现有属性进行修改？

1. 看到有chart.getNodeByKey方法，如何获取Key以便调用此方法？

1. 是否有推荐的做法？

**Comment by pearmini** [comment]
看看这里：https://g2.antv.antgroup.com/manual/extra-topics/event
